### DISCUSSION TITLE
Drag and frop Blocks between Builder components

### DISCUSSION THREAD

**Discussion by titantwentyone** [filamentphp/filament#7386] [Help]
First of all congrats on v3! Totally in love with Filament!

I raised a UI suggestion in a discussion last year about the ability to drag and drop blocks between builders on the same page. ([link](https://github.com/filamentphp/filament/discussions/4333#discussioncomment-3984364)) @zepfietje mentioned it was on the roadmap as a proposal. I did see it in there at one time but can't seem to find it now. Was this abandoned/rejected? I wondered if this was implemented in v3 but couldn't see anything in the new docs about this either. Appreciate any insight!

**Comment by zepfietje** [comment]
Hey, yeah I remember your proposal for this feature!
I probably removed it from the roadmap when doing a cleanup.
I'm not sure how much work it would be to support dragging and dropping blocks between builder items. Maybe @danharrin can shed some light on that?

**Reply by danharrin** [reply]
Honestly someone else can have a go, but I dont know how to approach that

**Comment by assertchris** [comment]
Anybody thought a bit more about this, yet? I have a couple compelling use-cases for this; and could quite possibly organise a bounty for someone to add the feature.
